geopolars/src/util.rs

```diff
@@ -13,3 +13,18 @@ pub fn iter_geom(series: &Series) -> impl Iterator<Item = Geometry<f64>> + '_ {
         Wkb(vec).to_geo().expect("unable to convert to geo")
     })
 }
+
+pub fn geom_at_index(series: &Series, index: usize) -> Result<Geometry<f64>> {
+    let chunks = series.list().expect("series was not a list type");
+    let row = chunks.into_iter().nth(index);
```

Comment:
I learned recently that `nth` iterates through (and discards) every element up to `n`. So it's `O(n)` not `O(1)`; and if we iterated over the series calling `geom_at_index` it would end up being `O(n^2)` I think.

I'm sure there _must_ be some polars method that accesses a value at an index in constant time. I looked through the [`Series`](https://pola-rs.github.io/polars/polars/series/struct.Series.html) and [`ChunkedArray`](https://pola-rs.github.io/polars/polars/chunked_array/struct.ChunkedArray.html) APIs quickly, though didn't immediately find the right method. I'm just a little worried that this `into_iter().nth()` is a bottleneck.